package.json

```diff
@@ -40,43 +40,43 @@
   "peerDependencies": {
     "@remix-run/react": "^1.0.0",
     "@remix-run/server-runtime": "^1.0.0",
-    "i18next": "^21.3.3",
+    "i18next": "^21.3.3 || ^22.0.3",
```

Comment:
For context, I used `^22.0.3` instead of `^22.0.0` since there were release 3 patch releases with fixes for the type definitions.